game/graphics/shaders.h

```diff
@@ -50,6 +50,17 @@ class Shaders {
 
     Tempest::RenderPipeline  tonemapping;
 
+    enum class FxaaPreset : int {
```

Comment:
Is there a need if `class`? In other part of code, preset seem to be used almost always as integer, with static_cast

Reply:
Used enum for readability, seems better than plain integers in the code section where fxaa's pso array is filled. 
Removed redundant prefix "FXAA_"